book.rb

```diff
@@ -0,0 +1,12 @@
+class Book
+  attr_accessor :title, :rentals
+
+  def initialize(title, _author)
```

Comment:
- [ ] Nice job with the Book class, but we are required to have `title` and `author` instance variables for the book. Kindly update the constructor accordingly. Please also specify `attr_accessor` for `author` as you did for title and rentals. ✍️

**Project requirement**
![image](https://github.com/zunairkhan811/OOP_School_Library/assets/17243325/c0ccebeb-2a55-4509-adeb-bc0141bf24af)

Reply:
Thanks! the change has been made